### PR TITLE
Check new defined uniqueness of poll.options

### PR DESCRIPTION
**Pull request by r-peschke**
[Checks the options in poll for uniqueness](https://github.com/OpenSlides/openslides-backend/issues/504) and [sets the values for absent options to -2](https://github.com/OpenSlides/openslides-backend/issues/493)

### CODE REVIEW

**Review by FinnStutzenstein — APPROVED** [review]

File: openslides_backend/action/actions/poll/create.py [hunk 1]
```diff
@@ -36,7 +37,6 @@ class PollCreateAction(CreateAction):
                 "type": "array",
                 "items": options_schema,
                 "minItems": 1,
-                "uniqueItems": True,
```
Comment (jsangmeister) [inline openslides_backend/action/actions/poll/create.py:39]: Isn't this check still right? Why did you remove it?

Comment (r-peschke) [reply openslides_backend/action/actions/poll/create.py:39]: It is still (or again) right, but not sufficient. The new check is stricter and therefore misleading. This check active we have 2 different error-messages for the same error from user/developer view.

Comment (jsangmeister) [reply openslides_backend/action/actions/poll/create.py:39]: Ah okay, I see. So for consistent error messages we remove it here.

**Review by jsangmeister — APPROVED** [review]
